src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write code here
+const tableHead = document.querySelector('table thead');
+const tableBody = document.querySelector('table tbody');
+const tableRows = [...document.querySelectorAll('table tbody tr')];
```

Comment:
you can use table selectors, like tBodies, tHead, rows, etc
https://learn.javascript.ru/dom-navigation#dom-navigation-tables